Modix.Services/Tags/TagInlineParsingHandler.cs

```diff
@@ -37,7 +37,14 @@ public async Task HandleNotificationAsync(MessageReceivedNotification notificati
             //TODO: Refactor when we have a configurable prefix
             if (message.Content.StartsWith('!')) { return; }
 
-            var match = _inlineTagRegex.Match(message.Content);
+            //Remove code blocks from the message we are processing
+            var content = Regex.Replace(message.Content, @"(`{1,3}).*(.\1)", string.Empty, RegexOptions.Singleline);
```

Comment:
Double check me on this, but I think this fails for this (admittedly somewhat pathological) example: https://regex101.com/r/wBQUtp/1

Thinking we might need to make this a [lazy match](https://docs.microsoft.com/dotnet/standard/base-types/quantifiers-in-regular-expressions#match-zero-or-more-times-lazy-match-) in the middle

```suggestion
            var content = Regex.Replace(message.Content, @"(`{1,3}).*?(.\1)", string.Empty, RegexOptions.Singleline);
```

Reply:
Yeah it's indeed failing for your example, i'm gonna check if it's working well with your suggestion